canonical-web-and-design/vanilla-framework
# Fix broken `snippet__line` outside of `snippet_block--numbered`

## Done

Fix `snippet__line` displaying incorrectly when used outside of `snippet_block--numbered`

Fixes #4357

## QA

- Open [demo](https://vanilla-framework-4363.demos.haus/docs/examples/patterns/code-snippet/numbered)
- It should display correctly
- inspect the page and take `--numbered` out of the parent element's class
- it shouldn't display any numbers

### Check if PR is ready for release

If this PR contains Vanilla SCSS code changes, it should contain the following changes to make sure it's ready for the release:

  - `Feature 🎁`, `Breaking Change 💣`, `Bug 🐛`, `Documentation 📝`, `Maintenance 🔨`.
  - if CSS class names are not changed it can be bugfix relesase (x.x.**X**)
  - if CSS class names are changed/added/removed it should be minor version (x.**X**.0)
  - see the [wiki for more details](https://github.com/canonical-web-and-design/vanilla-framework/wiki/Release-process#pre-release-tasks)


## Screenshots

Current behaviour:
![image](https://user-images.githubusercontent.com/11927929/158395290-cffa7477-8b5e-49b5-81fb-b26255181499.png)

After this is merged:
![image](https://user-images.githubusercontent.com/11927929/158395375-57be25b7-f3e9-499b-8d4b-888b4ddfc6a2.png)
